.travis.yml

```diff
@@ -1,11 +1,10 @@
 language: scala
 
 jdk:
-  - oraclejdk8
```

Comment:
Why remove that jdk version? In the `build.sbt` file you tell sbt to use jdk 8 for publishing, so we should keep that version. You can use `openjdk8` instead of `oraclejdk8`, if there is any issue with the latter.

Reply:
Thanks, that seems to have worked